### ISSUE TITLE
Ability to customize the ffmpeg command line

### ISSUE THREAD

**Issue by MaverickMSG** [ripieces/advancedfx#822]
As an HLAE user, I want maximum flexibility when outputting through ffmpeg.  Currently the beginning portions of the ffmpeg command line are hardcoded however this causes issues if an extremely complex processing pipeline is desired.  Specifically, the hardcoding of the command line has a filter defined in it, which will be overridden if another filter is added and prevents the 'copy' video codec from being utilized.

https://github.com/advancedfx/advancedfx/blob/main/shared/AfxOutStreams.cpp#L230 creates this:
/path/to/ffmpeg -f rawvideo -pixel_format bgr24 -loglevel repeat+level+warning -framerate 60 -video_size WIDTHxHEIGHT -i pipe:0 -vf setsar=sar=1/1,vflip

Ideal solution:
- Make the entire ffmpeg command line fully adjustable (through a variable), such that the user can define all arguments being passed to it.  The variable could have a default value of "-f rawvideo -pixel_format {IMGFMT} -loglevel repeat+level+warning -framerate {FPS} -video_size {WIDTH}x{HEIGHT} -i pipe:0 -vf setsar=sar=1/1,vflip" (or whatever is recommended) but could be fully modified.

**Comment by dtugend** [comment]
https://github.com/advancedfx/advancedfx/commit/09a8c2db502deeb0c09d971105e27185c26a7b0d
[AfxHooks_20230127T1748Z.zip](https://github.com/advancedfx/advancedfx/files/10521633/AfxHooks_20230127T1748Z.zip)

This includes already all other closed changes for v2.144.0 (so no vflip needed)!

Example settings:
```
mirv_streams settings add ffmpegEx ffEx "{QUOTE}{FFMPEG_PATH}{QUOTE} -f rawvideo -pixel_format {PIXEL_FORMAT} -loglevel repeat+level+warning -framerate {FRAMERATE} -video_size {WIDTH}x{HEIGHT} -i pipe:0 -vf setsar=sar=1/1 -c:v libx264 -preset slow -crf 22 {QUOTE}{AFX_STREAM_PATH}\\\\video.mp4{QUOTE}"
```

(Depending if console or cfg it might be `\\` instead of `\\\\`)

**Comment by dtugend** [comment]
https://github.com/advancedfx/advancedfx/commit/c39bd5f781a9a374461bd946ae62c13ca20dbaaf
[AfxHooks_20230128T1529Z.zip](https://github.com/advancedfx/advancedfx/files/10527592/AfxHooks_20230128T1529Z.zip)
